smarty/sampler_elemental.py

```diff
@@ -88,13 +88,13 @@ def __init__(self, molecules, basetypes_filename, initialtypes_filename, decorat
         This is just a proof of concept.  No scoring of molecular properties is performed.
 
         """
-        
-        
+
+
```

Comment:
Hmm, that's odd. Everyone who touched it was told to set that, but perhaps someone missed it.